harbor-registry.yaml

```diff
@@ -0,0 +1,62 @@
+package:
+  name: harbor-registry
+  version: 2.8.3
+  epoch: 0
+  description: An open source trusted cloud native registry project that stores, signs, and scans content (registry)
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - curl
+  environment:
+    DISTRIBUTION_DIR: "/usr/lib/go/src/github.com/docker/distribution"
+    CGO_ENABLED: "0"
+    GO111MODULE: "auto"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/distribution/distribution
+      tag: v${{package.version}}
+      expected-commit: 4772604ae973031ab32dd9805a4bccf61d94909f
+
+  - runs: |
+      # Fetch goharbor's redis patch
+      curl https://raw.githubusercontent.com/goharbor/harbor/main/make/photon/registry/redis.patch -o redis.patch
```

Comment:
Should we either vendor this patch in this repo or pin it by digest?

Reply:
Vendoring it here is probably way to go. Then we can just drop curl as an env dep